### ISSUE TITLE
Weight Uncertainty in neural network.

### ISSUE THREAD

**Issue by dlehdgus9887** [dlehdgus9887/paper-log-own#6]
## 어떤 내용의 논문인가요? 👋

기존의 parameter에다가 확률값을 더한다.

## Abstract (요약) 🕵🏻‍♂️

We  introduce  a  new,  efficient,  principled  andbackpropagation-compatible algorithm for learn-ing a probability distribution on  the weights ofa neural network, calledBayes by Backprop.  Itregularises  the  weights  by  minimising  a  com-pression cost, known as the variational free en-ergy or the expected lower bound on the marginallikelihood.   We  show  that  this  principled  kindof regularisation yields comparable performanceto  dropout  on  MNIST  classification.   We  thendemonstrate  how  the  learnt  uncertainty  in  theweights  can  be  used  to  improve  generalisationin non-linear regression problems, and how thisweight  uncertainty  can  be  used  to  drive  theexploration-exploitation  trade-off  in  reinforce-ment learning

## 이 논문을 읽어서 무엇을 배울 수 있는지 알려주세요! 🤔

이 논문을 제대로 읽었을 때 어떤 지식을 얻을 수 있을까요?

## 같이 읽어보면 좋을 만한 글이나 이슈가 있을까요?

https://daheekwon.github.io/MAB1/   (Bandit)
https://seunghan96.github.io/dl/stat/paper-Bayes_by_Backprop/
## 레퍼런스의 URL을 알려주세요! 🔗

markdown 으로 축약하지 말고, 원본 링크 그대로 그냥 적어주세요!
